src/ucs/sys/event_set.h

```diff
@@ -35,7 +35,8 @@ typedef enum {
     UCS_EVENT_SET_EVREAD         = UCS_BIT(0),
     UCS_EVENT_SET_EVWRITE        = UCS_BIT(1),
     UCS_EVENT_SET_EVERR          = UCS_BIT(2),
-    UCS_EVENT_SET_EDGE_TRIGGERED = UCS_BIT(3)
+    UCS_EVENT_SET_EDGE_TRIGGERED = UCS_BIT(3),
+    UCS_EVENT_SET_DUMMY          = UCS_BIT(4)
```

Comment:
this should not be here, it's outside the scope of event_set

Reply:
ok so will add a define to async.h

Reply:
yes, and it should be defined as 0